lib/TransportRPCServer.js

```diff
@@ -3,6 +3,12 @@
 const TransportRPC = require('./TransportRPC')
 
 class TransportRPCServer extends TransportRPC {
+  constructor (client, conf, Socket) {
+    super(client, conf)
+
+    this.Socket = Socket
```

Comment:
@robertkowalski isn't it better to move the `getSocket` inside the transport class?